src/main/java/org/zone/region/flag/player/falldamage/PlayerFallDamageFlag.java

```diff
@@ -0,0 +1,54 @@
+package org.zone.region.flag.player.falldamage;
+
+import org.jetbrains.annotations.NotNull;
+import org.jetbrains.annotations.Nullable;
+import org.zone.region.flag.Flag;
+import org.zone.region.flag.FlagType;
+import org.zone.region.flag.FlagTypes;
+import org.zone.region.flag.move.player.preventing.PreventPlayersFlag;
+import org.zone.region.group.key.GroupKey;
+import org.zone.region.group.key.GroupKeys;
+
+import java.util.Optional;
+
+public class PlayerFallDamageFlag implements Flag.TaggedFlag, Flag.Enabled, Flag.AffectsPlayer{
+
+    public @Nullable Boolean enabled;
```

Comment:
Tag flag cannot hold data

Reply:
meow ;c

Reply:
emm what about the cmds? boolean arg? enable.... setEnabled...

Reply:
No just remove the flag for disabled and add the flag for enabled